deploy-manage/deploy/cloud-on-k8s/_snippets/es-config.md

```diff
@@ -0,0 +1,39 @@
+Any setting defined in the `elasticsearch.yml` configuration file can also be defined for a set of {{es}} nodes in the `spec.nodeSets[?].config` section.
```

Comment:
```suggestion
{{es}} settings that are typically defined in the `elasticsearch.yml` configuration file can be specified for a set of nodes in the `spec.nodeSets[?].config` section of the {{es}} resource manifest.
```

Not sure, what do you think? I didn't like much using the past participle (`defined`).
I've also removed the `also`.